.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+<!-- The text between the arrows are comments - they will not be visible on your PR. -->
+<!-- To automatically tag this PR, add the uppercase label(s) surrounded by brackets below, for example: [LABEL] -->
```

Comment:
🤔

Reply:
What exactly is the issue? Emojis aren't really helpful.

Reply:
You don't have auto labeling functionality...

Reply:
the idea is probably to add them especially for the autolabeling functionality